app/src/test/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonsViewModelTest.kt

```diff
@@ -61,10 +103,70 @@ class LessonsViewModelTest {
         val first = randomTool("first", Tool.Type.LESSON, defaultOrder = 1, isHidden = false)
         val second = randomTool("second", Tool.Type.LESSON, defaultOrder = 2, isHidden = false)
 
+        languageFlow.value = Language(Locale.ENGLISH)
+        lessonsByLanguageFlow.value = listOf(second, first)
         viewModel.lessons.test {
-            lessonsFlow.value = listOf(second, first)
             runCurrent()
+            coVerify { toolsRepository.getLessonsFlowByLanguage(any()) }
```

Comment:
when verifying calls to methods you should be explicit with parameters. otherwise this test will still pass if the code is updated to send the wrong language to the method